src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,11 +40,22 @@ private void initializeSubsystems() {
      * initializes all commands
      */
     public void initializeCommands() {
+        Supplier<Double> x, y, rot;
+        if(RobotConstants.DriverConstants.SQUARED_CONTROLLER_DRIVING) {
+            x = () -> Math.pow(driverXbox.getLeftX(), 2);
+            y = () -> Math.pow(driverXbox.getLeftY(), 2);
+            rot = () -> Math.pow(driverXbox.getRightX(), 2);
+        } else {
+            x = driverXbox::getLeftX;
+            y = driverXbox::getLeftY;
+            rot = driverXbox::getRightX;
+        }
```

Comment:
have the logic be inside the command, maybe have it get a boolean of SQUARED_CONTROLLER_DRIVING and a defualt set to DriverConstants.SQUARED_CONTROLLER_DRIVING

also import static

Reply:
I disagree. I think that the command is only getting a double supplier. If you want to square it, do it in the supplier.

Reply:
but this is way too long